connectivity/manifests/allow-all-except-world.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: cilium.io/v2
+kind: CiliumNetworkPolicy
+metadata:
+  namespace: cilium-test
+  name: allow-all-except-world
+spec:
+  endpointSelector: {}
+  egress:
+  - toEntities:
+    - host
+    - remote-node
+    - cluster
+    - init
+    - health
+    - kube-apiserver
```

Comment:
This line breaks the tests in Cilium releases < 1.11:
```
  🔥 Running test allow-all-except-world: setting up test: applying network policies: policy application failed: CiliumNetworkPolicy.cilium.io "allow-all-except-world" is invalid: [spec.egress.toEntities: Unsupported value: "kube-apiserver": supported values: "all", "world", "cluster", "host", "init", "unmanaged", "remote-node", "health", "none", spec.ingress.fromEntities: Unsupported value: "kube-apiserver": supported values: "all", "world", "cluster", "host", "init", "unmanaged", "remote-node", "health", "none"]
```

Reply:
Might be best to open an issue for that.

Reply:
#719

Reply:
PR #720 is out to fix